INSTALL.md

```diff
@@ -860,6 +860,11 @@ This option will only produce a compliant FIPS provider if you have:
 
 Failure to do all of these will produce a non-compliant FIPS provider.
 
+### disable-fips186-4-ffc
```

Comment:
Please be consistent with the rest of the options.. 'disable-' is not used anywhere else.
i.e no-fips186-4-ffc